README.md

```diff
@@ -1,98 +1,120 @@
 [![GitHub CI](https://github.com/artefactual-labs/AIPscan/actions/workflows/test.yml/badge.svg)](https://github.com/artefactual-labs/AIPscan/actions/workflows/test.yml)
 [![codecov](https://codecov.io/gh/artefactual-labs/AIPscan/branch/main/graph/badge.svg?token=2RRFAM8P89)](https://codecov.io/gh/artefactual-labs/AIPscan)
 
-# AIPscan
 
-Collect repository-wide info about [Archivematica][am-1] Archival
-Information Packages (AIPs)
+# About
 
-## License
+AIPscan was developed to provide a more in-depth reporting solution for Archivematica users. It crawls METS files from AIPs in the Archivematica Storage Service to generate tabular and visual reports about repository holdings. It is designed to run as a stand-alone, add-on to Archivematica. It only needs a valid Storage Service API key to fetch source data.  
 
-Copyright Artefactual Systems Inc (2020).
-[Apache License Version 2.0](LICENSE)
+# License
 
-## Installation
+[Apache License Version 2.0](LICENSE)  
+Copyright Artefactual Systems Inc (2021)
 
-Below are the developer quickstart instructions. See [INSTALL](INSTALL.md) for production deployment instructions.
+# Contents
+* [Screenshots](#screenshots)
+* [Installation](#installation)
+* [Usage](#usage)
 
-### AIPScan Flask server
+# Screenshots
+
+## Example AIPScan fetch job
+
+![screencap1](screencaps/aipscan_fetch_job.png)
+
+## Viewing an AIP in AIPScan
+
+![screencap2](screencaps/aipscan_view_aip.png)
+
+## Selecting a report to run in AIPScan
+
+![screencap3](screencaps/aipscan_select_report.png)
```

Comment:
This screenshot is pretty out of date now - could we add a newer one in its place?

Reply:
Most definitely. Had same thought. I was just re-using what I had while I sorted out the text.